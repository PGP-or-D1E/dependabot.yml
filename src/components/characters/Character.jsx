import React from 'react';
import propTypes from 'prop-types';
import styles from './character.css';

const Character = ({ name, photoUrl }) => {
    
  return (
    <figure className={styles.Character}>
      <img src={photoUrl} />
      <figcaption>{name}</figcaption>
    </figure>
  );
};

Character.propTypes = {
  name: propTypes.string.isRequired,
  photoUrl: propTypes.string.isRequired
};

export default Character;
