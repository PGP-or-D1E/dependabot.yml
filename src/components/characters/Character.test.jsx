
import React from 'react';
import { render } from '@testing-library/react';
import Character from './Character';

describe('Character Snapshot', () => {
  it('matches a snapshot', () => {
    const { asFragment } = render(<Character 
      photoUrl="http://image.com"
      name="Billy"
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});
