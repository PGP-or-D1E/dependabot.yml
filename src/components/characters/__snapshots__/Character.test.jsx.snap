// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character Snapshot matches a snapshot 1`] = `
<DocumentFragment>
  <figure
    class="Character"
  >
    <img
      src="http://image.com"
    />
    <figcaption>
      Billy
    </figcaption>
  </figure>
</DocumentFragment>
`;
